pyproject.toml

```diff
@@ -1,11 +1,149 @@
+# Used for local development, not to be published.
```

Comment:
Should we add this? https://docs.astral.sh/uv/guides/publish/#preparing-your-project-for-packaging:~:text=If%20you%20have,be%20accidentally%20published.

Reply:
You mean the classifier? It's not needed because it's not a package.